### ISSUE TITLE
Alton Taing - Lightning Presentation

### ISSUE THREAD

**Issue by altaing** [ShawnTylerSchwartz/eeb-c177-w20#73]
https://youtu.be/1Cjp2YDSGfk

**Comment by omerlavian20** [comment]
Fantastic project! Using data science to determine which factors affect disease outcomes is an **extremely important** application of this methodology, so good on you for doing that!

I have some questions for you. Firstly, how did you generate such beautiful graphs in Python? I thought you could only make such pretty graphs in R. Secondly, did you use R for data visualization? If so, what did you use it for that you could not achieve in Python?

**Comment by soniavsd** [comment]
Great presentation! The way you explained your code and reasonings were very clear and thorough, especially your code demonstration! I agree with Omer, your graphs look really nice in python. I wasn't able to make x and y labels, so its very impressive!

**Comment by goharmihranian** [comment]
This presentation/research is great because its so practical! Any research that has to do with the heart is useful. I agree with both Sonia and Omer; your graphs in Python are beautiful! Tell us your secret! I also love how you did inputs for everything. It makes things look so much cleaner and well organized. Great work!

**Comment by jpocon** [comment]
Hi Alton, great talk! I have to agree with everyone that your Python plots rival those of R's capabilities. Personally, I have tried to make visually appealing plots in Python to avoid code-switching to R, but the number of lines used in R was always fewer than those in Python. What modules do you use? After jumping into ggplot in class, do you still prefer Python?

**Comment by haonguyen318** [comment]
Hi Alton! I really like the visualization for your graph. The colors on the graph made it easy to see a trend on the graph. It was also very helpful that you explained what the r-square and p-value means in terms of the data. Your code demonstration was also very clear and I liked how you worked out how you created your plots. Overall great job on your presentation!

**Comment by estmendoza** [comment]
Hi Alton! I really liked that you made a key for user input. It seems like that makes it easier for the user to interact with the program. I'm also really impressed by how clean your graphs look. What do you think was the most challenging part of creating them?

**Comment by dylanreadel** [comment]
I am interested in the potential of expanding your programming concepts beyond heart disease data to other deadly diseases in this country and worldwide. For your plot that visualizes cholesterol against resting blood pressure, I noticed there was an outlier around a cholesterol level of 550. I am curious about whether you are able to implement limits in the code to remove outliers to gain more accurate results and plots. Overall, I enjoyed your presentation!
